docs/_data/table/en/cluster-management-configuration.yml

```diff
@@ -19,7 +19,7 @@ alluxio.integration.mesos.role:
 alluxio.integration.mesos.secret:
   'Secret token for authenticating with Mesos.'
 alluxio.integration.mesos.user:
-  'The Mesos user for the Alluxio Mesos Framework.'
+  'The Mesos user for the Alluxio Mesos Framework. Defaults to the current user'
```

Comment:
Are the yml files generated? This is missing the period.

Reply:
These are autogenerated, its unclear to me why the period didn't get picked up.

Reply:
I copy pasted them from `PropertyKey` since autogen would have its own set of problems.